### PR TITLE
DietPi-Cron | Disable cron.minutely by default

### PR DESCRIPTION
**Pull request by MichaIng**
**Status**: Ready

**Commit list/description**:
- Place readme.txt into /etc/cron.minutely/. Due to file ending, this is not executed by cron/run-parts.
- Disable it by default to reduce unneeded cron executions and logs.
- Disable it with patch_file, if no file was placed inside by user.

### CODE REVIEW

File: dietpi/patch_file [hunk 1]
```diff
@@ -1014,6 +1014,9 @@ _EOF_
 			# - qBitTorrent: https://github.com/Fourdee/DietPi/issues/1957
 			chown -R qbittorrent:dietpi /home/qbittorrent &> /dev/null
 			#-------------------------------------------------------------------------------
+			# - Disable cron.minutely, if unused to reduce cron executions and logs
+			(( $(find /etc/cron.minutely/ | wc -l) > 2 )) || sed -i '\|cron\.minutely|s|^\*/[0-9]*[[:blank:]]|#*/0 |' /etc/crontab
```
Comment (Fourdee) [inline dietpi/patch_file:1018]: Should be ```>= 2``` (or ```> 1```)?

Empty directory
```
root@DietPi:~# find /etc/cron.minutely/ | wc -l
1
```

One file in directory:
```
root@DietPi:~# find /etc/cron.minutely/ | wc -l
2

root@DietPi:~# ls /etc/cron.minutely/
test

```

**Review by Fourdee — COMMENTED** [review]
Needs checking.

File: dietpi/patch_file (hunk 1 above)
Comment (MichaIng) [reply dietpi/patch_file:1018]: @Fourdee 
The new readme.txt will be inside as well 😄: https://github.com/Fourdee/DietPi/blob/cron.minutely/rootfs/etc/cron.minutely/readme.txt

Comment (Fourdee) [reply dietpi/patch_file:1018]: @MichaIng 

Even with new glasses, I still couldn't see that lol

**Pull request merged**